packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -29,6 +32,11 @@ function BottomSheetSettings( {
 	settings,
 	...props
 } ) {
+	const colorSettings = {
+		colors: useEditorFeature( 'color.palette' ) || settings.colors,
+		gradients: useEditorFeature( 'color.gradients' ) || settings.gradients,
```

Comment:
`useEditorFeature` needs to be renamed after https://github.com/WordPress/gutenberg/pull/31587